packages/webdriver-utils/src/driver.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'node-fetch';
+
+export default class Driver {
+  constructor(sessionId, executorUrl) {
+    this.sessionId = sessionId;
+    this.executorUrl = executorUrl;
+  }
+
+  async getCapabilites() {
```

Comment:
use cache class from appium js sdk here. It becomes pretty expensive to fetch capabilities per screenshot [ ideally you only want to do it once per `sessionId` ]